server/cluster/cluster.go

```diff
@@ -490,11 +483,30 @@ func (c *RaftCluster) SetStorage(s *core.Storage) {
 	c.storage = s
 }
 
+// GetBasicCluster returns the basic cluster.
+// There is no need a lock since it won't changed.
```

Comment:
Written in `RaftCluster.InitCluster`

Reply:
ditto

Reply:
After discussing with @HunDunDM, I'm going to raise another PR to do it.